Detectors/PHOS/workflow/src/ClusterizerSpec.cxx

```diff
@@ -43,8 +43,8 @@ void ClusterizerSpec::init(framework::InitContext& ctx)
     ccdbManager.setURL(o2::base::NameConf::getCCDBServer());
     LOG(info) << " set-up CCDB " << o2::base::NameConf::getCCDBServer();
 
-    BadChannelsMap* badMap = ccdbManager.get<o2::phos::BadChannelsMap>("PHS/BadMap");
-    CalibParams* calibParams = ccdbManager.get<o2::phos::CalibParams>("PHS/Calib");
+    BadChannelsMap* badMap = ccdbManager.get<o2::phos::BadChannelsMap>("PHS/Calib/BadMap");
+    CalibParams* calibParams = ccdbManager.get<o2::phos::CalibParams>("PHS/Calib/CalibParams");
```

Comment:
above you change to this: https://github.com/AliceO2Group/AliceO2/blob/9e546ae0d3a1199026eb910316b7bee97cd84bae/Detectors/PHOS/calib/src/PHOSBadMapCalibDevice.cxx#L137-L145
and here you again use PHS/Calib/BadMap. Could you cross-check?

Reply:
Thank you for careful reading. Idea is to store intermediate maps from different methods in PHS/BadMap and manually combine them to final map stored in PHS/Calib/BadMap.

Reply:
Ingredients of bad map will be run automatically. Combination will be done once per modification of electronics ~ as in Run2 per period or few periods.